float64[] waypoint_Q1
float64[] waypoint_Q2
float64[] waypoint_Q3
float64[] waypoint_Q4
float64[] waypoint_Q5
float64[] waypoint_Q6
